pallets/liquidity-proxy/src/liquidity_aggregator/selector.rs

```diff
@@ -69,10 +68,8 @@ where
         &mut self,
         source: LiquiditySourceType,
         discrete_quotation: DiscreteQuotation<AssetIdOf<T>, Balance>,
```

Comment:
Maybe call this method `add_source_unchecked`?

Reply:
For what?

Reply:
To show that this function does not perform any checks before adding liquidity and it's caller's responsobility to perform any checks

Reply:
It makes sense only when we have another "checked" function.